[{
	"raw": {},
	"required": [
		"Filename",
		"Size"
	],
	"package": "libboost-exception1.83-dev",
	"source": "boost1.83",
	"version": "1.83.0-4.2~bpo12+1",
	"section": "libdevel",
	"priority": "optional",
	"architecture": "amd64",
	"depends": [
		"libboost1.83-dev (= 1.83.0-4.2~bpo12+1)"
	],
	"conflicts": [
		"libboost-exception1.50-dev",
		"libboost-exception1.52-dev",
		"libboost-exception1.53-dev",
		"libboost-exception1.54-dev",
		"libboost-exception1.55-dev",
		"libboost-exception1.57-dev",
		"libboost-exception1.58-dev",
		"libboost-exception1.60-dev",
		"libboost-exception1.61-dev",
		"libboost-exception1.62-dev",
		"libboost-exception1.63-dev",
		"libboost-exception1.65-dev",
		"libboost-exception1.67-dev",
		"libboost-exception1.70-dev",
		"libboost-exception1.71-dev",
		"libboost-exception1.74-dev",
		"libboost-exception1.80-dev",
		"libboost-exception1.81-dev"
	],
	"installedSize": 2146,
	"maintainer": "Debian Boost Team <team+boost@tracker.debian.org>",
	"description": "library to help write exceptions and handlers",
	"homepage": "http://www.boost.org/libs/exception/",
	"filename": "pool/main/b/boost1.83/libboost-exception1.83-dev_1.83.0-4.2~bpo12+1_amd64.deb",
	"size": 230212,
	"sha256": "053783e2c7d82a82b33432e03443e88865ff58713238549b8bcb6a1fb92d61f5",
	"descriptionMd5": "384c2c28f8227e2f185f0778df8c37fc"
}]